src/portal-components/_DetailsPage.scss

```diff
@@ -81,9 +81,7 @@ $svg-icon-height: 30px;
   }
 
   &.tab-layout {
-    .component-container {
```

Comment:
This did not play well with the sticky menu (component content was moved up, right next to the tab).  
component-container sets the layout, but tab-layout is responsible for moving the tabs up (into the header card area).